### DISCUSSION TITLE
Dracula theme for grub not working

### DISCUSSION THREAD

**Discussion by RvDekens** [dracula/dracula-theme#853] [Q&A] [answered]
Hello, I try to install Dracula theme for grub but it's not working. I've clone the repo to _/home/Documents/temp_, copied the dracula folder to _/boot/grub/themes_ and add the line "GRUB_THEME='/boot/grub/themes/dracula/theme.txt" in the grub file (_/etc/default/_).
I've updated the grub config file using `sudo update-grub` and got that messages :

![image](https://user-images.githubusercontent.com/131585155/233832272-1c08b598-41f0-4e7b-b0ab-7db378bb2426.png)

Does anyone had this problem ? How can I solve it ?
Thx for your answers

**Comment by luxonauta** [comment, marked as answer]
Hi 👋🏻, thank you for your interest in providing this feedback. ✨

But there is a better place to share the same, which is the project repository, where you can talk to the theme maintainers.

✅ [Clicking here](https://github.com/dracula/grub/issues/new), you can open an Issue on it.

**Reply by RvDekens** [reply]
Ok, thank you.
I've noticed that it's not only the theme Dracula that's not working but any theme I try to use...
I will ask in another forum.
